internal/rbd/rbd_util.go

```diff
@@ -64,7 +64,7 @@ const (
 	encryptionMetaKey = ".rbd.csi.ceph.com/encrypted"
 
 	// go-ceph will provide rbd.ImageOptionCloneFormat
-	imageOptionCloneFormat = librbd.RbdImageOption(12)
+	imageOptionCloneFormat = librbd.RbdImageOption(12) // nolint:gomnd // number represents unique key.
```

Comment:
This is included in go-ceph v0.4. Drop the const and replace `imageOptionCloneFormat` with `rbd.ImageOptionCloneFormat` in the code.